.gitignore

```diff
@@ -5,3 +5,6 @@
 /bin
 /dist
 /config.toml
+/gui/.tmp
+/gui/yarn.lock
```

Comment:
nice!  do you think we should move to yarn for our other javascript-dependent projects?

Reply:
I started using `yarn` recently because `npm` became unusable for me (`npm install` takes ages). `yarn` is compatible with `npm` except (lock files: `yarn.lock` vs `npm_shrinkwrap.json`) so there shouldn't be a problem using it now in other projects.